mux/src/localpane.rs

```diff
@@ -1055,7 +1055,7 @@ impl LocalPane {
             // Since windows paths typically start with something like C:\,
             // we cannot simply stick `localhost` on the front; we have to
             // omit the hostname otherwise the url parser is unhappy.
```

Comment:
this comment is not longer in sync with the implementation. I feel reasonably confident that we can remove this comment and not have to directly test this change on windows, because we are already using `from_directory_path` in `wezterm set-working-directory`.

So, let's remove this comment!

Reply:
Done!